apps/web/components/atoms/heading/heading.tsx

```diff
@@ -0,0 +1,31 @@
+import type { HTMLAttributes, ReactNode } from 'react';
+import cx from 'clsx';
+import type { TextSize } from 'atoms/text/text';
+
+export type HeadingTag = keyof Pick<
+	JSX.IntrinsicElements,
+	'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
+>;
+
+type HeadingProps = {
+	children: ReactNode;
+	tag: HeadingTag;
+	size: TextSize;
+} & HTMLAttributes<HTMLHeadingElement>;
+
+export const Heading = ({ children, tag, size, ...rest }: HeadingProps) => {
+	const Tag = tag;
```

Comment:
Zacytuję @Survikrowa: "taka mała pierdółka, że możesz już zrobić rename w momencie destrukturyzacji prospów :D
`tag: Tag` mamy juz w jakims komponencie tak zrobione".
```ts
export const Heading = ({ children, tag: Tag, size, ...rest }: HeadingProps) => {
...
}
```

Reply:
Słuszna uwaga, poprawiłem to również w komponencie Text